src/js/modules/plant/object.js

```diff
@@ -2,18 +2,15 @@ import lodash from 'lodash';
 import { View } from '../../core';
 import PlantViewTools from '../plant/tools';
 import Const from '../../const';
+import moveableComponent, { MOVE_END } from '../components/moveable';
```

Comment:
I'm not a fan of this solution with `default` and others here. I will be more elegant (I think so) to export only all symbols explicitly named.